db/xsql/src/fetch.rs

```diff
@@ -15,8 +15,9 @@ use ckb_types::core::{
     TransactionBuilder, TransactionView, UncleBlockView,
 };
 use ckb_types::packed::{
-    Byte32, Byte32Vec, CellDep, CellInput, CellInputBuilder, CellOutput, CellOutputBuilder,
-    OutPointBuilder, ProposalShortIdVec, UncleBlockBuilder,
+    Byte32, Byte32Vec, BytesVec, CellDepVec, CellInput, CellInputBuilder, CellOutput,
```

Comment:
All the types that are packed by molecule should be used as `packed::Type`. The reason for this is that it is easy to distinguish between the molecule type and the original type.